src/App.js

```diff
@@ -17,13 +17,13 @@ class App extends React.Component {
           {/* <Header /> */}
           <Switch>
             <Route path="/" exact component={Warehouse} />
-            <Route path="/warehouse" exact component={Warehouse} />
-            <Route path="/warehouse/:id" exact component={WarehouseDetails} />
+
+            <Route path="/warehouse" component={Warehouse} />
+            <Route path="/warehouse/:id" component={Warehouse} />
```

Comment:
Carried over {WarehouseDetails} from previous version

```suggestion
            <Route path="/warehouse/:id" exact component={WarehouseDetails} />
```